src/components/RankedList.tsx

```diff
@@ -19,6 +19,7 @@ interface RankedListProps {
     bgColor: string;
   };
   rankColor: string;
+  headingLink: string;
```

Comment:
```suggestion
  headingLink?: string;
```

Maybe have it optional?